test/features/firestore-consent-handlers.scenario.feature

```diff
@@ -0,0 +1,46 @@
+Feature: pisp-demo-server server
+
+Scenario: Create Consent
+  Given pisp-demo-server server
+  When I create a new Consent
+  Then the status should be updated and a request id should be assigned
+
+Scenario: Create Consent With Defined Status
+  Given pisp-demo-server server
+  When I create a Consent with a defined status
```

Comment:
```suggestion
  When I create a Consent with a "defined" status
```
take a look for reusable snippets and passing parameters: https://github.com/bencompton/jest-cucumber/blob/HEAD/docs/ReusingStepDefinitions.md